proposals/0229-simd.md

```diff
@@ -608,3 +599,42 @@ have been replaced with the "normal" arithmetic operations, matching `+`, `-`, `
 5. We have removed several operations whose naming was not totally satisfactory. We
 expect to add those operations and more in future additive proposals; only the types are
 necessary for ABI stability.
+
+6. Updates based on discussion on Swift-Evolution and with core team: I have eliminated
+`_`-prefixed protocol requirements, to make it somewhat more obvious how user types
+can be made to conform. This required some renaming to make the function of these
+associatedtypes more clear. I have also systematically removed `Vector` from the naming
+scheme, in order to remove confusion with C++-style vectors or "mathematical" vector
+structures that might be added at some future point. The resulting names are generally
+shorter, which is an nice benefit, and they now have a uniform `SIMD` prefix.
+
+  Protocols renamed:
```

Comment:
[You must include a blank line before your table in order for it to correctly render](https://help.github.com/articles/organizing-information-with-tables/).

PS: please reconsider `SIMDVector`, if `SIMDMatrix` is also possible.

Reply:
The discussion with core was that we would use SIMD4x4 or similar when this happens.